src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/StatusAlert.jsx

```diff
@@ -64,25 +99,13 @@ export default function StatusAlert({ appointment, facility }) {
       </InfoAlert>
     );
   }
-  if (isPastAppointment && featureAfterVisitSummary) {
+  if (isPastAppointment && !!featureAfterVisitSummary) {
     return (
       <>
         <p className="vads-u-font-size--base vads-u-font-weight--bold vads-u-font-family--sans vads-u-margin-top--2 vads-u-margin-bottom--0">
-          This appointment occurred in the past.
+          This appointment happened in the past.
         </p>
-        {avsLink && (
-          <va-link
-            text="Go to after-visit summary"
-            href={appointment.avsPath}
-            data-testid="after-vist-summary-link"
-            onClick={() =>
-              recordEvent({
-                event: `${GA_PREFIX}-after-visit-summary-link-clicked`,
-              })
-            }
-          />
-        )}
-        <br />
+        {avsLink && <>{handleAvsLink()}</>}
```

Comment:
@vbahinwillit - `handleAvsLink` is not a component. It's a function that returns jsx but because it's called as a function: `handleAvsLink()`, it's not considered a component. I will change the name to `displayAvsLink` to make it clear that it's not a click handler.

Reply:
@simiadebowale - I thought any function that returns jsx is considered a component.

Reply:
@vbahinwillit - That's a common misconception. Here's a very helpful [article](https://dev.to/igor_bykov/react-calling-functional-components-as-functions-1d3l#:~:text=gets%20re%2Dmounted.-,Conclusion,and%20can%20have%20a%20state.) with more details

Reply:
@vbahinwillit - looking at the rule proposal you linked. It's been open since 2016. So I believe it's safe to say there's no timeline for that right now.

Reply:
@simiadebowale - That link came from the article you originally link. I guess we will have to agree to disagree on this. I do agree that there is no definitive decision of this but it does seems to be convention to me. The context for how you are using the function implies component.